src/adhocracy_pcompass/adhocracy_pcompass/resources/subscriber.py

```diff
@@ -0,0 +1,72 @@
+"""Sync external ressources with elastic search."""
+
+import re
+import requests
+import logging
+
+from substanced.util import find_service
+from pyramid.traversal import find_interface
+
+from adhocracy_core.interfaces import IResourceCreatedAndAdded, search_query
```

Comment:
This is not in pep8 but still a good rule (and I think it is also in our project-internal style guide).